lib/beetle/base.rb

```diff
@@ -31,6 +31,11 @@ def current_port
       @server =~ /:(\d+)$/ ? $1.to_i : 5672
     end
 
+
+    def connection_options_for_server(server)
```

Comment:
The authority over the configuration is still in the client. This merely delegates it and makes it easier accessible for descendents of base.